pantab/tests/test_roundtrip.py

```diff
@@ -50,11 +50,20 @@ def test_multiple_tables(df, tmp_hyper, table_name, table_mode):
         expected = pd.concat([expected, expected]).reset_index(drop=True)
 
     # some test trickery here
-    if not isinstance(table_name, tab_api.TableName) or table_name.schema_name is None:
-        table_name = tab_api.TableName("public", table_name)
+    if not isinstance(table_name, TableName) or table_name.schema_name is None:
+        table_name = TableName("public", table_name)
 
-    assert set(result.keys()) == set(
-        (table_name, tab_api.TableName("public", "table2"))
-    )
+    assert set(result.keys()) == set((table_name, TableName("public", "table2")))
     for val in result.values():
         assert_roundtrip_equal(val, expected)
+
+
+def test_roundtrip_with_external_hyper_process(df, tmp_hyper):
+    # By passing in a pre-spawned HyperProcess, one can e.g. avoid creating a log file
```

Comment:
Can you add to this test a check that a log file is not created? You should be able to use the built in pathlib module to check for that

Reply:
done